WordPress/gutenberg
# Search block: switch interactivity to the Interactivity API

## What?

Fixes #53292

## Why?

To add consistency to all Core blocks that needs frontend interactivity.

## How?


## Testing Instructions




1. Add a search block in any page, template.
2. Set it as button only.
3. Check that transitions on click are still working.

### Testing Instructions for Keyboard


## Screenshots or screencast
